ament_cobra/ament_cobra/main.py

```diff
@@ -124,7 +120,7 @@ def main(argv=sys.argv[1:]):
 
     # If the user has provided a valid ruleset with --ruleset, add it
     if args.ruleset in rulesets:
-        cmd.extend(['-f', args.ruleset])
+        cmd.extend(['-f', args.ruleset if args.ruleset != 'autosar' else 'C++/autosar'])
```

Comment:
I am tempted to mount a case for being as transparent to cobra as possible so that ament_cobra and direct cobra usage remain as consistent as possible but I'm not strongly opposed to this so I'll leave it as discretionary feedback only.

Reply:
@mjeronimo I wanted to raise this but I don't consider it a blocker. I'll invite you to merge this one as you finished it up! Thanks!

Reply:
I was kinda ambivalent about this one and opted for the ease-of-use approach. However, I didn't think of consistency with the direct cobra usage. I'll update to use 'C++/autosar' as recommended.